api/src/routes/index.ts

```diff
@@ -279,6 +291,21 @@ router
   .route("/api/travel-authorization-action-logs")
   .get(TravelAuthorizationActionLogsController.index)
 
+// TravCom Integration
+router
+  .route("/api/trav-com/accounts-receivable-invoices")
+  .get(TravComIntegration.Controllers.AccountsReceivableInvoicesController.index)
+router
+  .route("/api/trav-com/accounts-receivable-invoices/:accountsReceivableInvoiceId")
+  .get(TravComIntegration.Controllers.AccountsReceivableInvoicesController.show)
+
+router
+  .route("/api/trav-com/accounts-receivable-invoice-details")
+  .get(TravComIntegration.Controllers.AccountsReceivableInvoiceDetailsController.index)
+router
+  .route("/api/trav-com/accounts-receivable-invoice-details/:accountsReceivableInvoiceDetailId")
+  .get(TravComIntegration.Controllers.AccountsReceivableInvoiceDetailsController.show)
```

Comment:
Should I still keep these?

Reply:
Kept list accounts-receivable-invoice-details as it lets me display the number of records that will be synced in the UI. Deleted all the rest.